### ISSUE TITLE
Pinch/Spread Gesture to adjust Grid Size

### ISSUE THREAD

**Issue by Gymcap** [ApolloTVofficial/kamino#74]
See [Poweramp Music Player](https://play.google.com/store/apps/details?id=com.maxmpz.audioplayer) (Google Play) for a good example of this functionality.

Expected implementation:

# Primary:  
• User is browsing Movies/TV Shows with 3 columns.  
• User performs a pinch gesture to adjust to 4 columns. (Up to 5 or 6 would cover landscape devices pretty well)  
• User performs a spread gesture to adjust back to 3 columns.

• After every change, this value should be stored with its corresponding orientation somewhere in the app data.

# Secondary:  
• Upon using a spread gesture with the 3 column view, the grid will transform into a list view.                    

# TL;DR
Pinch = +1 column  
Spread = -1 column until list view

**Comment by BradB111** [comment]
Very nice idea, will also need to implement an option in settings for devices without a touchscreen if this is to be added for compatibility.

**Comment by kevincosmos** [comment]
Duplicate of #40. Closing.
